### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Vhirttuegaming** [3kh0/ext-remover#818] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How do you remove google account on skiovox helper?

**Comment by 3kh0** [comment, marked as answer]
you just signout

**Reply by Vhirttuegaming** [reply]
What does the button of four arrows pointing away do?

**Reply by Su4o** [reply]
Picture? (use your phone or something else)

**Reply by Vhirttuegaming** [reply]
Its like a compass icon on the top right of the skiovox helper
